docs/cloud/cloud-troubleshooting/issue-serverless-limitations.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Serverless limitations
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 4
+---
+{% include /serverless/serverless-preview-warning.md %}
+
+# Issue - Known limitations of serverless
```

Comment:
Headings need to be self-explanatory to non-experts. "Serverless" isn't clear on its own.

```
# Issue - Known limiatations of Serverless Cloud Databases
```